openapi.yml

```diff
@@ -1902,7 +1865,85 @@ components:
           nullable: true
         prison:
           $ref: "#/components/schemas/Prison"
-
+    PrisonerOffenderSearch:
+      type: object
```

Comment:
Why PrisonerOffenderSearch and ProbationOffenderSearch are two objects if they are the same?

Reply:
you're right, I needed to rekick the pipeline, I will squish them together now